app/surveys/mci_0213.py

```diff
@@ -0,0 +1,20 @@
+import copy
+
+from app.surveys.mci_0215 import mci_0215
+
+eq_id = "1"
+form_type = "0213"
+
```

Comment:
Would this be easier the other way round? I.e. add the additional question on 0215?